src/index.html

```diff
@@ -17,6 +17,43 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="nav">
+        <div class="logo">
+          <img
+            src="images/logo.png"
+            alt="logo Moyo"
+          />
+        </div>
+        <nav>
+          <ul class="link">
+            <li class="box box-1">
+              <a
+                href="index.html"
+                class="is-active"
```

Comment:
Fix class everywhere
```suggestion
        <nav class="nav">
          <ul class="nav__list">
            <li class="nav__item">
              <a
                href="index.html"
                class="nav__link is-active"
```